scripts/validatePackage.js

```diff
@@ -37,7 +37,8 @@ execOrError('npm run build');
 // Create a tarball, and move it to a known path (drop the version from the pkg name)
 console.log('PREP: packaging');
 execOrError(`npm pack --pack-destination ${JSON.stringify(WORK_DIRECTORY)}`);
-execOrError('mv *.tgz tolgee-cli.tgz', { cwd: WORK_DIRECTORY });
+const moveCommand = process.platform === 'win32' ? 'move' : 'mv';
+execOrError(`${moveCommand} *.tgz tolgee-cli.tgz`, { cwd: WORK_DIRECTORY });
```

Comment:
It might be best to use a method from `fs` here to be honest

Reply:
I was thinking about that too, but `fs` doesn't support glob patterns like `*.tgz` as far as I know.

Do you think it would still be worth to change this line to use `fs` with a `for` loop for example?

Reply:
Oh that's right, yeah

It might be possible to just extract it from the output of npm pack and just feed this to later on (instead of the constant tarball dest set) 🤔

Reply:
I fully removed the renaming of the tarball by using the `version` field from the `package.json` to get the file's name.